prusti-encoder/src/encoders/type/predicate.rs

```diff
@@ -1,5 +1,6 @@
 use prusti_rustc_interface::{
-    abi,
+    // abi,
+    target::abi,
```

Comment:
Why the `// abi` comment? Also in `type/domain.rs`.

Reply:
They moved stuff around, the relevant types are no longer exported as part of `abi` but `target::abi`.